src/transaction/types.ts

```diff
@@ -15,6 +15,11 @@ import type {
 
 export type Call = { to: Hex; data?: Hex; value?: bigint };
 
+type TransactionButtonOverride = {
+  text?: string;
+  onClick?: (receipt?: TransactionReceipt) => void;
+};
```

Comment:
Define in `TransactionButton` if this is used for a singular purpose

Reply:
it will be exported